### PR TITLE
support /fapi/v3/positionRisk

### LINKED ISSUE #602: Request upgrade NewGetPositionRiskService V2 => V3
/fapi/v2/positionRisk upgrade to /fapi/v3/positionRisk

### PR DESCRIPTION
**Pull request by xyq-c-cpp**
1. support /fapi/v3/positionRisk, close #602

### CODE REVIEW

File: v2/futures/client.go [hunk 1]
```diff
@@ -525,7 +525,10 @@ func (c *Client) NewGetBalanceService() *GetBalanceService {
 	return &GetBalanceService{c: c}
 }
 
-// NewGetPositionRiskService init getting position risk service
+func (c *Client) NewGetPositionRiskV2Service() *GetPositionRiskV2Service {
+	return &GetPositionRiskV2Service{c: c}
+}
+
```
Comment (adshao) [inline v2/futures/client.go:531]: Is there any reason to keep this V2 service? In my opinion, we only need the latest version of each service.

**Review by adshao — CHANGES_REQUESTED** [review]

File: v2/futures/client.go (hunk 1 above)
Comment (xyq-c-cpp) [reply v2/futures/client.go:531]: Binance still provides this interface, so I have kept it. Of course, I can also keep only the V3 version

Comment (adshao) [reply v2/futures/client.go:531]: > Binance still provides this interface, so I have kept it. Of course, I can also keep only the V3 version @adshao

OK, that's fine, we can keep it as long as binance provides both v2/v3 interfaces.

**Review by adshao — APPROVED** [review]

File: v2/futures/client.go (hunk 1 above)
Comment (vlw) [reply v2/futures/client.go:531]: @xyq-c-cpp @adshao 
The second version should definitely be kept, as it allows you to view the leverage and margin type for each position at once in one query, while in version 3 you can't do that - you will have to make a query /fapi/v1/symbolConfig for each open position to view the current leverage, which will require much more queries and will use up the rate limit much faster.

**Pull request merged**